controller/embedded/api-schema/entities/service-edge-router-policy.post.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$id": "http://ziti-edge.netfoundry.io/schemas/service-edge-router-policy.post.json",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "name"
+  ],
+  "properties": {
+    "name": {
+      "$ref": "definitions.all.json#/definitions/name"
+    },
+    "semantic": {
+      "default": null,
+      "$ref": "definitions.all.json#/definitions/semantic"
```

Comment:
Even though `default` here is impotent, would null be a valid value for semantic? I assume it defaults to the `all` semantic.

Reply:
Yes, if default worked, the default would be all. But since it doesn't, it's just trying to indicate that you can pass in nothing or null and then the model code will default it to any for you.